src/partials/_header.hbs

```diff
@@ -1,16 +1,13 @@
 <header class="header">
-  <nav class="nav">
-    <img class="logo--lg" src="../icons/ame-logo-lg.svg">
-    <img class="logo--sm" src="../icons/ame-logo-sm.svg">
-
-    <ul class="nav__list left">
-      <li class="list__item"><a href="#" class="item__link">Products</a></li>
-      <li class="list__item"><a href="#" class="item__link">FAQ</a></li>
-    </ul>
-
-    <ul class="nav__list right">
-      <li class="list__item"><a href="#" class="item__link">Retailers</a></li>
-      <li class="list__item"><a href="#" class="item__link">About</a></li>
-    </ul>
-  </nav>
+	<h1 class="site-title">
```

Comment:
Ok, not sure why but I think GitHub is adding all that extra space. In Brackets it looks just fine.